src/cljs/rems/fields.cljs

```diff
@@ -141,7 +141,7 @@
                                                (str (field-name opts) "-error"))
                            :max-length max-length
                            :class (when validation "is-invalid")
-                           :defaultValue value
+                           :value value
```

Comment:
`date-field` and `option-field` still use `defaultValue`. is that ok? I'm fine with this being in a follow-up PR.

Reply:
I'll fix it here.